bin/pinto

```diff
@@ -17,6 +17,8 @@ use warnings;
 my $home_var = 'PINTO_HOME';
 my $home_dir = $ENV{$home_var};
 
+$ENV{GNUPGHOME} = $ENV{PINTO_GNUPGHOME} if $ENV{PINTO_GNUPGHOME};
```

Comment:
Originally had this in the `BUILD` for `Pinto::Verifier` so it would only be set when a verifier was instantiated. On reflection I thought that was silly. Would local in package scope on `Pinto::Verifier` work/be okay? Or should I push it down to the 2 routines that call `Module::Signature`